vcsclient/azurerepos_test.go

```diff
@@ -352,10 +352,12 @@ func TestAzureReposClient_DeleteWebhook(t *testing.T) {
 
 func TestAzureReposClient_SetCommitStatus(t *testing.T) {
 	ctx := context.Background()
-	client, cleanUp := createServerAndClient(t, vcsutils.AzureRepos, true, "", "unsupportedTest", createAzureReposHandler)
+	commitHash := "86d6919952702f9ab03bc95b45687f145a663de0"
+	expectedUri := "/_apis/ResourceAreas/commitStatus"
+	client, cleanUp := createServerAndClient(t, vcsutils.AzureRepos, true, nil, expectedUri, createAzureReposHandler)
 	defer cleanUp()
-	err := client.SetCommitStatus(ctx, 1, owner, repo1, "", "", "", "")
-	assert.Error(t, err)
+	err := client.SetCommitStatus(ctx, 1, owner, repo1, commitHash, "", "", "")
+	assert.NoError(t, err)
```

Comment:
Let's also add negative test:
```suggestion
	assert.NoError(t, err)
	
	badClient, badClientCleanup := createBadAzureReposClient(t, []byte{})
	defer badClientCleanup()
	err = badClient.SetCommitStatus(ctx, 1, owner, repo1, commitHash, "", "", "")
	assert.Error(t, err)
```